test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/ConverterTests.cs

```diff
@@ -21,14 +25,48 @@ public class ConverterTests
 
         // there is rounding
         [InlineData(1239, "1.24K")]
+        [InlineData(9056, "9.06K")]
+        [InlineData(19_056, "19.1K")]
+        [InlineData(19_556, "19.6K")]
+        [InlineData(99_056, "99.1K")]
+        [InlineData(99_556, "99.6K")]
 
-        [InlineData(123400, "123K")]
-        [InlineData(1234000, "1.23M")]
-        [InlineData(12340000, "12.3M")]
-        [InlineData(1234000000, "1.23G")]
-        public void DownloadCountToStringTest(int num, string expected)
+        [InlineData(123_400, "123K")]
+        [InlineData(899_560, "900K")]
+        [InlineData(998_999, "999K")]
+        [InlineData(999_001, "999K")]
+        [InlineData(999_560, "999K")]
+        [InlineData(1_234_000, "1.23M")]
+        [InlineData(1_299_560, "1.3M")]
+        [InlineData(9_995_650, "10M")]
+        [InlineData(12_340_000, "12.3M")]
+        [InlineData(19_991_250, "20M")]
+        [InlineData(19_999_850, "20M")]
+        [InlineData(99_156_050, "99.2M")]
+        [InlineData(99_999_650, "100M")]
+        [InlineData(999_560_050, "999M")]
+        [InlineData(999_999_050, "999M")]
+        [InlineData(1_000_000_001, "1B")]
+        [InlineData(1_234_000_000, "1.23B")]
+        [InlineData(2_100_999_050, "2.1B")]
+        [InlineData(9_234_000_000, "9.23B")]
+        [InlineData(9_999_900_000, "10B")]
+        [InlineData(99_199_199_650, "99.2B")]
+        [InlineData(99_999_999_650, "100B")]
+        [InlineData(999_999_999_650, "999B")]
+        [InlineData(9_999_999_999_650, "10T")]
+        [InlineData(19_999_999_999_050, "20T")]
+        [InlineData(99_999_999_999_650, "100T")]
+
+        // there is localization
+        [InlineData(1939, "1.94K", "en-US")]
+        [InlineData(1939, "1,94K", "da-DK")]
+        [InlineData(9456, "9.46K", "en-US")]
+        [InlineData(9456, "9,46K", "da-DK")]
```

Comment:
>[InlineData(9456, "9,46K", "da-DK")] [](http://example.com/codeflow?start=8&length=36)

Could we also add tests for B vs G, since Andy pointed out that G will be used for locales following the metric system?

Reply:
@aortiz-msft
I thought about it but I believe it's available after localization team make translation for each different cultures. @dominoFire Please correct me if I'm wrong.

Reply:
Also, since NuGet.Client's localization happens in a different repo, we can't have unit tests that work the same and still pass both locally (that doesn't run MicroBuild's localization task) and on CI (which does run MicroBuild's localization task).

I believe that if we migrated to the same localization system/format that the rest of the dotnet org uses, then local builds would include it.